optimism.tokenlist.json

```diff
@@ -847,6 +847,28 @@
       "symbol": "WETH",
       "decimals": 18,
       "logoURI": "https://ethereum-optimism.github.io/logos/WETH.png"
+    },
+	{
+      "chainId": 42,
+      "address": "0x001dedC8c67EF6D1f4BB66FB28581d466650Cd76",
+      "name": "Dentacoin",
+      "symbol": "DCN",
+      "decimals": 0,
+      "logoURI": "https://ethereum-optimism.github.io/logos/logo.svg",
```

Comment:
This should also be the `dcn-l2-logo-big.png` file but can you please rename the image to just `dcn.png`